app/state/reducers/searchFiltersReducer.js

```diff
@@ -1,12 +1,16 @@
+import moment from 'moment';
 import { handleActions } from 'redux-actions';
 
-const initialState = {
-  search: '',
-};
+function getInitialState() {
+  return {
+    date: moment().format('YYYY-MM-DD'),
```

Comment:
Do you think we should always save dates as strings in the store? And in general where do you think we should be using Moments/strings in the code?